lms/djangoapps/course_home_api/progress/views.py

```diff
@@ -223,18 +231,26 @@ def get(self, request, *args, **kwargs):
             'status_date': verification_status['status_date'],
         }
 
+        access_expiration = get_access_expiration_data(request.user, course_overview)
+
+        # User locale settings
+        user_timezone_locale = user_timezone_locale_prefs(request)
+        user_timezone = user_timezone_locale['user_timezone']
```

Comment:
I thought you don't need these since it is wrapped into the course_metadata call now?

Reply:
Yes good catch